include/unk_02031AF0.h

```diff
@@ -0,0 +1,10 @@
+#ifndef POKEHEARTGOLD_UNK_02031AF0_H
+#define POKEHEARTGOLD_UNK_02031AF0_H
+
+#include "save.h"
+
+u16 sub_02031AF0(); //02031AF0
```

Comment:
This comment is redundant

Also, any functions that don't take in any parameters should just have `void`:
```suggestion
u32 sub_02031AF0(void);
```